### PR TITLE
ref task/10826 modify built-in auth

### PR DESCRIPTION
**Pull request by sirnguyenhuudat**
https://edu-redmine.sun-asterisk.vn/issues/10826

### CODE REVIEW

**Comment by sirnguyenhuudat** [comment]
ready

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+
+    <title>{{ config('app.name', 'Laravel') }}</title>
+
+    <!-- Scripts -->
+    <script src="{{ asset('js/app.js') }}" defer></script>
+
+    <!-- Fonts -->
+    <link rel="dns-prefetch" href="//fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet" type="text/css">
```
Comment (BunnyPi04) [inline resources/views/layouts/app.blade.php]: tải font về sử dụng nhé

File: app/Models/User.php [hunk 2]
```diff
@@ -54,6 +54,47 @@ public function comments()
...
+    }
+
+    //    /**
+//     * @param string|array $roles
+//     */
+    public function authorizeRoles($roles)
+    {
+        if (is_array($roles)) {
+            return $this->hasAnyRole($roles) ||
+                abort(401, 'This action is unauthorized.');
```
Comment (BunnyPi04) [inline app/Models/User.php]: i18n

File: resources/views/home/layouts/modal.blade.php [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <!-- Modal -->
 <div class="ms_register_popup">
-    <div id="myModal" class="modal  centered-modal" role="dialog">
+    <!-- Register Modal-->
+    <div id="modalRegister" class="modal  centered-modal" role="dialog">
```
Comment (BunnyPi04) [inline resources/views/home/layouts/modal.blade.php]: `modal  centered-modal`
dư 1 space

**Pull request merged**